pages/business/dashboard.tsx

```diff
@@ -12,6 +12,47 @@ import { AddCard } from '~ui/blocks/AddCard'
 import { BusinessDataModal } from '~ui/modals/BusinessDataModal'
 import { BusinessDeleteModal } from '~ui/modals/BusinessDeleteModal'
 
+const CompanyCard = ({ company, openModal }) => {
```

Comment:
You could move this component into a new file: `~ui/blocks/CompanyCard.tsx`. There are already a bunch of other single-use components inside the `blocks` directory.